Penteract/Assets/Scripts/PlayerController.cpp

```diff
@@ -368,27 +404,27 @@ void PlayerController::Update() {
 		playerOnimaru.Update(useGamepad);
 	}
 
-	if (playerFang.isAlive && playerOnimaru.isAlive) {
-		CheckCoolDowns();
-		UpdatePlayerStats();
+	//if (playerFang.isAlive && playerOnimaru.isAlive) {
```

Comment:
this line should be commented?
If ins not necessary, should it be removed?

Reply:
You're absolutely right